docs/release-notes/highlights-1.1.0.asciidoc

```diff
@@ -1,2 +1,73 @@
 [[release-highlights-1.1.0]]
 == 1.1.0 release highlights
+
+[float]
+[id="{p}-110-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 1.1.0 of {n}. See <<release-notes-1.1.0>> for the full list of changes.
+
+[float]
+[id="{p}-110-remote-clusters"]
+==== Remote cluster support
+
+Establishing uni-directional connections from ECK-managed Elasticsearch clusters to other Elasticsearch clusters for link:https://www.elastic.co/guide/en/elasticsearch/reference/current/xpack-ccr.html[cross-cluster replication] and link:https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-cross-cluster-search.html[cross-cluster search] purposes is now easy to achieve through a <<{p}-remote-clusters,few additions to the YAML manifest>>. This feature is available for all managed clusters running Elasticsearch 7.6 or higher with an Enterprise or Enterprise trial licence.
+
+
+[float]
+[id="{p}-110-declarative-user-management"]
+==== Declarative user management
+
+File realm users and roles are now <<{p}-users-and-roles,configurable using Kubernetes secrets>>. This enables consistent and auditable user management practices and GitOps use cases.
+
+
+[float]
+[id="{p}-110-enhanced-validation"]
+==== Enhanced validation
+
+Validation has been extended to more ECK resource types to catch invalid configurations during admission. Enabling the <<{p}-webhook,validating webhook>> will extend the capabilities of ECK to catch most common configuration issues during update operations as well.
+
+
+[float]
+[id="{p}-110-rbac"]
+==== Enforce RBAC on associations
+
+Access control policies can be enforced on cross-namespace Elasticsearch associations created by Kibana and APM Server instances. This feature is disabled by default to avoid breaking existing deployments. See <<{p}-restrict-cross-namespace-associations>> for more information.
+
+
+[float]
+[id="{p}-110-kibana-encryption-key"]
+==== Stable Kibana encryption key
+
+Unless the user has <<{p}-kibana-scaling,provided their own `xpack.security.encryptionKey`>>, the operator will generate a key that persists across restarts and upgrades. This prevents existing sessions from being invalidated due to a Kibana restart.
```

Comment:
That's  a bugfix, is it worth highlighting this here?

Reply:
It was mentioned in the blog post so I thought it would be odd to not include here. Happy to remove it if you think it should go.

Reply:
Ah ok let me check the blog post first :-)

Reply:
I guess its a bit less prominent in the blog post.

Reply:
OK, I will remove it.